scripts/build/build_examples.py

```diff
@@ -131,10 +131,17 @@ def ValidateTargetNames(context, parameter, values):
     help=(
         'Set pigweed command launcher. E.g.: "--pw-command-launcher=ccache" '
         'for using ccache when building examples.'))
+@click.option(
```

Comment:
We specifically tried to NOT have this. This creates too many variations.

If you need this, you can use straight `gn gen`.